src/main/kotlin/com/example/mykku/feed/FeedService.kt

```
@@ -46,9 +51,34 @@ class FeedService(
+
+        val feedTagTitles = feedTags.map { it.title }.toSet()
+        val activeContestsWithTags = contestReader.getActiveContestsWithAllTags()
+
+        activeContestsWithTags
+            .filter { (_, requiredTags) ->
+                requiredTags.isNotEmpty() && feedTagTitles.containsAll(requiredTags)
+            }
+            .forEach { (contest, _) ->
+                try {
+                    contestParticipationWriter.participateViaFeed(member, contest, feed)
+                } catch (e: ContestException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)